packages/gallery/src/components/item/media/mediaItem.tsx

```diff
@@ -71,13 +71,15 @@ export default function MediaItem<T extends Record<string, any>>(props: MediaPro
     if (utils.isSSR()) {
       return false;
     }
-
     if (
       playTrigger === GALLERY_CONSTS[optionsMap.behaviourParams.item.video.playTrigger].HOVER ||
       playTrigger === GALLERY_CONSTS[optionsMap.behaviourParams.item.video.playTrigger].AUTO
     ) {
       return true;
-    } else if (clickAction === GALLERY_CONSTS[optionsMap.behaviourParams.item.clickAction].NOTHING) {
+    } else if (
+      clickAction === GALLERY_CONSTS[optionsMap.behaviourParams.item.clickAction].NOTHING ||
+      clickAction === GALLERY_CONSTS[optionsMap.behaviourParams.item.clickAction].MAGNIFY
```

Comment:
if click action is NOTHING than why do we want to have the full video render? it's that means it's thumbnail?